Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -57,7 +57,7 @@ function vtkResliceCursorWidget(publicAPI, model) {
     // at least completely cover the viewed region
     const planeSource = vtkPlaneSource.newInstance();
 
-    if (viewType === ViewTypes.XZ_PLANE) {
+    if (viewType === ViewTypes.YZ_PLANE) {
```

Comment:
according to the below code, it should be XZ_PLANE

Reply:
XZ plane means that the plane is constructed with X and Z axis. So, the constructed plane is Y plane. We want the associated Y normal.

Reply:
isn't it what the code does ?

Reply:
Ok. I didn't check the correct lines.
But I inverted these two lines because there was a mistake when we moved from medical view name to plane name.
cf https://github.com/Kitware/vtk-js/pull/1620
I just fixed it.